### ISSUE TITLE
Install template failed after upgrade to 0.22.16

### ISSUE THREAD

**Issue by conechan** [saojs/sao#88]
```bash
cone-rmbp :: ~/Projects/test $ sao template 123
> Installing template-template with yarn...
yarn install v1.5.1
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
info Lockfile not saved, no dependencies.
✨  Done in 0.10s.
error template was not found at ~/.sao/packages/node_modules/template-template
cone-rmbp :: ~/Projects/test $ sao --version                                                              1 ↵
0.22.16
```

The log is as above. I am afraid some issues was introduced by the semi update.

**Comment by egoist** [comment]
Fixed in 0.22.17

**Comment by conechan** [comment]
Thanks!
